### DISCUSSION TITLE
Missing from cards

### DISCUSSION THREAD

**Discussion by webstmatt** [ExperienceLovelace/ha-floorplan#170] [Q&A] [answered]
I've installed ha-floorplan in HACS, and added the js file to resources.

But when I click on "Add Card" in Lovelace edit mode, floor plan is not available in the list. Is there a reason why this is the case?

**Comment by exetico** [comment, marked as answer]
You need to add the YAML manually, as it's mentioned in our guide 👍. (If I remember correctly)

**Reply by OzGav** [reply]
Here’s one you prepared earlier 😁
https://community.home-assistant.io/t/floorplan-now-available-as-a-lovelace-card/115489/323?u=ozgav

**Reply by exetico** [reply]
Thanks! I was on the mobile, but would like to help in the situation. I guess I should pro. a new written installation-guide.

**Reply by webstmatt** [reply]
Oh i see, thank you!
